frontend/src/components/Map/index.js

```diff
@@ -23,617 +32,640 @@ export default function Map() {
       zoom: state.zoom,
     });
 
-    const stateToFIPS = {
-      AK: "02",
-      AL: "01",
-      AR: "05",
-      AS: "60",
-      AZ: "04",
-      CA: "06",
-      CO: "08",
-      CT: "09",
-      DC: "11",
-      DE: "10",
-      FL: "12",
-      GA: "13",
-      GU: "66",
-      HI: "15",
-      IA: "19",
-      ID: "16",
-      IL: "17",
-      IN: "18",
-      KS: "20",
-      KY: "21",
-      LA: "22",
-      MA: "25",
-      MD: "24",
-      ME: "23",
-      MI: "26",
-      MN: "27",
-      MO: "29",
-      MS: "28",
-      MT: "30",
-      NC: "37",
-      ND: "38",
-      NE: "31",
-      NH: "33",
-      NJ: "34",
-      NM: "35",
-      NV: "32",
-      NY: "36",
-      OH: "39",
-      OK: "40",
-      OR: "41",
-      PA: "42",
-      PR: "72",
-      RI: "44",
-      SC: "45",
-      SD: "46",
-      TN: "47",
-      TX: "48",
-      UT: "49",
-      VA: "51",
-      VI: "78",
-      VT: "50",
-      WA: "53",
-      WI: "55",
-      WV: "54",
-      WY: "56",
-    };
-
-    var data = {};
-    const url = "https://covidtracking.com/api/states";
-    fetch(url)
-      .then((resp) => resp.json())
-      .then(function (dat) {
-        data = dat;
-      })
-      .catch(function (error) {
-        console.log(error);
-      });
-
+    const worldData = fetchCovidData(dataScope.WORLD);
+    const usStatesData = fetchCovidData(dataScope.US_STATES);
     map.on("load", function () {
-      // Add source for state polygons hosted on Mapbox, based on US Census Data:
-      // https://www.census.gov/geo/maps-data/data/cbf/cbf_state.html
-      map.addSource("states", {
-        type: "vector",
-        url: "mapbox://mapbox.us_census_states_2015",
-      });
+      addWorldLayer(map, worldData);
+      addUSStatesLayer(map, usStatesData);
+      // addUSCountyLayer(map);
+    });
+  }, []);
 
-      // exclude states outside the 50 states
-      const expression = ["match", ["get", "STATE_ID"]];
-      // var maxValue= data.reduce((max,b) => Math.max(max, b.death), data[0].death);
-      const normalizerValue = 5000;
-      data.forEach(function (row) {
-        var stateID = row.state;
-        if (stateID in stateToFIPS) {
-          var red = (row.death / normalizerValue) * 255;
-          var color = "rgba(" + red + ", " + 0 + ", " + 0 + ", 1)";
-          expression.push(stateToFIPS[stateID], color);
-        }
-      });
+  return (
+    <div className={styles.root}>
+      <div className={classNames(styles.fill)} id="map"></div>
+      {!draggable && <div className={classNames(styles.fill, styles.mask)} />}
+    </div>
+  );
+}
+
+async function fetchCovidData(scope) {
+  return await api(`data/` + scope, {
```

Comment:
I'm not going to block the PR with this, but for future reference, in JS strings are not added (sure, it works, but it's semantically undesirable and saying "they're not added" helped me memorize it back when I learnt this). [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) are the usual preferred sugar.

```javascript
return await api(`data/${scope}`, {
// ...
```

It's trivial here, but it can easily get out of hand 😛